beancount_reds_importers/libreader/csv_multitable_reader.py

```diff
@@ -56,7 +53,8 @@ def read_raw(self, file):
         return etl.fromcsv(file.name)
 
     def is_section_title(self, row):
-        return len(row) == 1
```

Comment:
I'd rather keep this as it was, to provide a reasonable starting point for this method.

Reply:
done in https://github.com/redstreet/beancount_reds_importers/pull/27/commits/921a7518a1aa29f127594c5a4c6b87141d915d50 , just left an additional comment gesturing towards using a custom override if needed